Services/Migration/test/ilServicesMigrationSuite.php

```diff
@@ -0,0 +1,27 @@
+<?php declare(strict_types=1);
+
+/******************************************************************************
```

Comment:
See my comment above, the licese header differs from the definition in our `Rector`.

Reply:
Thank you for the hint. I didn't know where to pick it. Hopefully now it is correct.